### PR TITLE
rsx: Allow to configure vblank rate

### PR DESCRIPTION
**Pull request by elad335**
Laying the groundwork for emulation speed modifiers.

Other improvements:
* Fixed an accuracy regression from #5719 (b8fb83c4483d). (recheck period before sleeping)

### CODE REVIEW

File: rpcs3/Emu/RSX/RSXThread.cpp [hunk 1]
```diff
@@ -456,37 +456,57 @@ namespace rsx
...
-						});
-
-						thread_ctrl::notify(*intr_thread);
+						start_time += period_time;
+
+						sys_rsx_context_attribute(0x55555555, 0xFED, 1, 0, 0, 0);
+
+						if (isHLE)
+						{
+							vblank_count.release(vblank_count.load() + 1);
```
Comment (kd-11) [inline rpcs3/Emu/RSX/RSXThread.cpp]: Why not just post/pre-increment?

Comment (elad335) [reply rpcs3/Emu/RSX/RSXThread.cpp]: its an atomic variable now its not the same.

Comment (kd-11) [reply rpcs3/Emu/RSX/RSXThread.cpp]: why do we not want a locked instruction though? Does that not defeat the purpose of making it atomic in the first place?

Comment (elad335) [reply rpcs3/Emu/RSX/RSXThread.cpp]: its for memory ordering and seq cst store before launching the thread.

Comment (elad335) [reply rpcs3/Emu/RSX/RSXThread.cpp]: And this guarentees single copy atomicity of load + release, even tho on x86 its placibo.

Comment (elad335) [reply rpcs3/Emu/RSX/RSXThread.cpp]: Used `vblank_count.raw()++;` for now.

Comment (elad335) [reply rpcs3/Emu/RSX/RSXThread.cpp]: Actually for pure correctness it needs se cst ordering so the value will be stored immediatly and wont get delayed.

File: rpcs3/Emu/Cell/Modules/cellGcmSys.cpp [hunk 2]
```diff
@@ -857,7 +857,7 @@ u64 cellGcmGetVBlankCount()
 {
 	cellGcmSys.trace("cellGcmGetVBlankCount()");
 
-	return rsx::get_current_renderer()->vblank_count;
+	return rsx::get_current_renderer()->vblank_count.load();
```
Comment (kd-11) [inline rpcs3/Emu/Cell/Modules/cellGcmSys.cpp]: Is this necessary?

Comment (elad335) [reply rpcs3/Emu/Cell/Modules/cellGcmSys.cpp]: Not really

Comment (kd-11) [reply rpcs3/Emu/Cell/Modules/cellGcmSys.cpp]: Why add it then? It may seem minor but picture a PR with dozens of such changes.

**Pull request merged**